run.sh

```diff
@@ -79,6 +79,9 @@ case "$deployment_action" in
     "deploy_airship")
         deploy_airship
         ;;
+    "update_airship_osh")
```

Comment:
shouldn't you do two steps instead?

I mean if there are real reasons you want to expose this feature, it might be worth exposing to the end users in a simple way:
setup_hosts -> setup_airship_undercloud -> setup_airship_openstack?

Reply:
Currently deploy_airship is idempotent so it can be executed again. In a way, logic is already separated into steps as internally it calls 2 separate roles for setting up ucp (undercloud) and osh (overcloud)

Reason for adding this is to quickly test and apply changes in osh site. The users mentioned in this can be devs.

If we would have been executing plays directly (e.g. ansible-playbook -i <inventory> <high_level play_name>) , then this would have been a separate play name instead of target for run.sh script.
I was thinking having different target based on most common use cases. From user perspective, deploy_airship and then updating existing osh/overcloud site seems more common pattern. May be add-compute another target as that can be another use-case.